pkg/controller/direct/certificatemanager/dnsauthorization_controller.go

```diff
@@ -107,9 +95,10 @@ func (m *model) AdapterForObject(ctx context.Context, reader client.Reader, u *u
 	}
 
 	// Get location
-	// TODO: Add location when field is added
-	// location := obj.Spec.Location
-	location := "global"
+	location := obj.Spec.Location
+	if location == "" {
+		location = "global"
```

Comment:
it would be nice to v(2) log that we are defaulting this but that can be follow on.